lib/plugins/acts_as_customizable/lib/acts_as_customizable.rb

```diff
@@ -48,7 +48,7 @@ def self.included(base)
         end
 
         def available_custom_fields
-          CustomField.where("type = '#{self.class.name}CustomField'").sorted.to_a
+          @available_custom_fields ||= CustomField.where("type = '#{self.class.name}CustomField'").sorted
```

Comment:
There are different kinds of custom fields (Issue, Project, User, Version? ...)  Can this level of caching possibly go wrong in the case various kinds of custom fields are queried on the same request? (This question is somewhat similar to what Stephan mentions but cross types of custom fields)

Reply:
But in the end is this changed line really necessary, won't the RAILS active record QUERY cache work on to avoid repeted calls here?

Reply:
I have reverted this, this was only instance-level memorization. I guess, it won't have too much problem.